api/src/main/java/openconsensus/trace/BlankSpan.java

```diff
@@ -89,6 +89,11 @@ public void addLink(Link link) {
     Utils.checkNotNull(link, "link");
   }
 
+  @Override
+  public void setSpanName(String spanName) {
```

Comment:
yep. It is aligned with the changes we made in C# and issue: https://github.com/census-instrumentation/opencensus-specs/issues/217